pkg/auth/providers/keycloakoidc/keycloak_provider.go

```diff
@@ -87,10 +85,10 @@ func (k *keyCloakOIDCProvider) toPrincipal(principalType string, acct account, t
 		displayName = acct.Username
 	}
 	princ := v3.Principal{
-		ObjectMeta:  metav1.ObjectMeta{Name: k.GetName() + "_" + principalType + "://" + strconv.Itoa(acct.ID)},
+		ObjectMeta:  metav1.ObjectMeta{Name: k.Name + "_" + principalType + "://" + strconv.Itoa(acct.ID)},
 		DisplayName: displayName,
 		LoginName:   acct.Username,
-		Provider:    k.GetName(),
+		Provider:    k.Name,
```

Comment:
k.GetName() returns the Name itself, so why this change?

Reply:
I changed it here to match how I am using Name and Type in the oidc_provider. But I do like the idea of using GetName() in the specific providers so I will revert this.